test/loadTest/orion_updates_ngsiv2.jmx

```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Script should use PATCH instead of POST

This assumes that the entities are already in the DB previous to scriopt execution. That could be mentioned in the README.md as prerequisite to use the script.